package.json

```diff
@@ -5,8 +5,11 @@
   },
   "scripts": {
     "build": "nuxt build",
+    "build:production": "nuxt build --dotenv .env.production",
```

Comment:
I think it's easier to maintain just have both `.env.production` and `.env.develop` or `.env` as files instead of just dev as hardcode constant